be/test/exprs/vectorized/math_functions_test.cpp

```diff
@@ -1302,8 +1302,8 @@ TEST_F(VecMathFunctionsTest, TrigonometricFunctionTest) {
     }
 
     {
-        std::vector<double> result_expect =
-            {std::tanh(-1), std::tanh(0), std::tanh(1), std::tanh(3.1415926), std::tanh(30)};
+        std::vector<double> result_expect = {std::tanh(-1), std::tanh(0), std::tanh(1), std::tanh(3.1415926),
+                                             std::tanh(30)};
         std::unique_ptr<FunctionContext> ctx(FunctionContext::create_test_context());
         ColumnPtr result = MathFunctions::tanh(ctx.get(), columns);
         auto v = ColumnHelper::cast_to<TYPE_DOUBLE>(result);
```

Comment:
remove these unrelated change

Reply:
ok, this change due to run clang-format.sh, can be restored since it none of this PR's business, changed.